### ISSUE TITLE
wpfterminal.codeplex.com

### ISSUE THREAD

**Issue by s2quake** [AurelienRibon/blog#1]
I leave here because there is no way to contact you. Can you transfer the source from https://wpfterminal.codeplex.com/ to github? 
Although the name is the same, I don't know if you're building this library. If yes, I would appreciate it if I could transfer the bug to this source so I could use bug fixes for this source.

**Comment by AurelienRibon** [comment]
Of course. That's a really old project I made when I was doing some C#. I was young and the code quality is young too. I'll try to bring it back here.

**Comment by s2quake** [comment]
Thank you

**Comment by s2quake** [comment]
I sent you an email a month ago.
But you didn't reply to the email.
I would appreciate it if you could check my email and reply.
